### PR TITLE
Add desktop implementation of url_launcher from flutter/plugins

### LINKED ISSUE #224: [flutter/plugins] Support url_launcher
It would be great if we could add support for this plugin:
https://github.com/flutter/plugins/tree/master/packages/url_launcher

Thanks!

### PR DESCRIPTION
**Pull request by stuartmorgan**
Sets up a new structure and README for landing implementations of
plugins from the flutter/plugins repository before they are ready to
be officially hosted (e.g., while plugin APIs stabilize).

Using that structure, adds an initial implementation of url_launcher
for macOS and Linux as a proof of concept.

Fixes #224 (except for Windows, for which we have #105)

### CODE REVIEW

File: plugins/flutter_plugins/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+These implementations exist here only as a temporary solution to use these
+plugins while the plugin APIs on each desktop platform stabilize enough that
+they can move to an official location and be distributed as normal Flutter
+plugins.
+
+## Using These Plugins
+
+For these plugins, the Dart code comes from the official plugin, so you
+must include that in your `pubspec.yaml` as well. For instance, to use
+url\_launcher on desktop, you would include:
```
Comment (franciscojma86) [inline plugins/flutter_plugins/README.md]: url_launcher*

Comment (stuartmorgan) [reply plugins/flutter_plugins/README.md]: Resolved offline; this was referring to the `\` which is intentional escaping.

**Comment by lesnitsky** [comment]
awesome! thanks for letting me know!

**Comment by lesnitsky** [comment]
@stuartmorgan just tried it, works perfectly

**Pull request merged**